fluentd/tests/test_reverse_proxy.sls

```diff
@@ -0,0 +1,20 @@
+test_nginx_is_installed:
+    testinfra.package:
+      - name: nginx
+      - is_installed: True
+
+test_nginx_running:
+  testinfra.service:
+    - name: nginx
+    - is_running: True
+    - is_enabled: True
+
+test_nginx_listening_http:
+  testinfra.socket:
+    - name: tcp://0.0.0.0:80
+    - is_listening: True
+
+test_nginx_listening_https:
+  testinfra.socket:
+    - name: tcp://0.0.0.0:443
+    - is_listening: True
```

Comment:
Should we be testing other states too, like if the config file was written, and if the cert/key matches up?